tcav/model.py

```diff
@@ -23,6 +23,7 @@
 import numpy as np
 import six
 import tensorflow as tf
+from google.protobuf import text_format
```

Comment:
need to add pip dep on protobuf for this, unless we already have it

Reply:
Added it. I also tested it locally using "pip install protobuf" after uninstalled the protobuf dependency, and made sure things still work. Not sure why I had the protobuf dependency initially when I was writing the test.